app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -23,6 +23,8 @@ class Scene extends React.Component {
     mutation: PropTypes.func.isRequired,
   };
 
+  imageCache = {};
```

Comment:
Since its a separate idea, can we pull image caching into its own commit?

Reply:
Where is this used?

Reply:
https://github.com/mojotech/helios2/pull/128/commits/fc0a94a5df76082bb99c0bc477ffee01ea723713#diff-ed8e5fea165eae9a5f4092f20661d47fR164